main.go

```diff
@@ -3,30 +3,80 @@ package main
 import (
 	"log"
 	"net"
+	"strings"
 
 	"git.icyphox.sh/taigobot/db"
 	"git.icyphox.sh/taigobot/plugins"
 	"github.com/dgraph-io/badger/v3"
 	"gopkg.in/irc.v3"
 )
 
+// (nojusr): A simple check func to find out if an irc message
+// is supposed to be a CTCP message
+func isCTCPmessage(m *irc.Message) bool {
+	// NOTE(nojusr): CTCP messages are identified by their first character byte
+	// being equal to 0x01.
+	if m.Params[1][0] == 0x01 {
+		return true
+	}
+	return false
```

Comment:
since it's a boolean, it can be simplified as:
```suggestion
	return m.Params[1][0] == 0x01
```